swagger/swaggerDoc.yaml

```diff
@@ -0,0 +1,849 @@
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"user\": {\n    \"email\": \"nya@hoppe.io\",\n\
+            \    \"password\": \"password\"\n  }\n}"
+      responses:
+        200:
+          description: "Status 200"
+  /api/web_app_config:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    put:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+  /app/{imageName}:
+    get:
+      description: "imageName=nope.jpg"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    parameters:
+    - name: "imageName"
+      in: "path"
+      required: true
+      type: "string"
+  /{jsonFile}:
+    post:
+      description: "jsonFile=csp_reports.json"
+      produces:
+      - "application/json"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+          examples:
+            application/json: "{\n  \n}"
+        400:
+          description: "Status 400"
+          schema:
+            type: "object"
+            example: "{\n  \"problem\": \"Crashed while parsing report\"\n}"
+    parameters:
+    - name: "jsonFile"
+      in: "path"
+      required: true
+      type: "string"
+  /tos_update:
+    get:
+      description: "???\nwhat is a tos_update??"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+  /api/corpuses/{IntId}:
+    get:
+      summary: "Corpuse or Corpuses"
+      description: "* with IntId-Variable:  get one corpuse.\n* no IntId-Variable:\
+        \ get all corpuses"
+      produces:
+      - "application/json"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/corpuse"
+    parameters:
+    - name: "IntId"
+      in: "path"
+      required: true
+      type: "string"
+  /api/device:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            $ref: "#/definitions/device"
+    post:
+      consumes: []
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+          properties:
+            name:
+              type: "string"
+          example: "{\n  \"name\": \"maculate-moon-349\"\n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            $ref: "#/definitions/device"
+    delete:
+      description: "IMHO - cant work - how the server should know which device has\
```

Comment:
We know which device to delete because:

1. The user is logged in (you _must_ be logged in to use this endpoint)
2. Users have _exactly 1 device_ (this might change later, but right now you're only allowed to have one) 

Therefore, if you are logged in and want to delete your device, we delete the only device that the account has.